tensorflow_lite_support/tools/Build_TensorFlow_Lite_Support_Libraries_with_Bazel_and_Colab.ipynb

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        library = '//tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/text:task-library-tex'
</pre><p> </p><p>Should be task-library-text?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tflite-support/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
OK